packages/rn-tester/package.json

```diff
@@ -10,8 +10,19 @@
     "directory": "packages/rn-tester"
   },
   "dependencies": {
+    "anser": "^1.4.9",
```

Comment:
@suminkimm why do we need all these new dependencies?

The idea is to not use third party dependencies inside `rn-tester`, which is why you see only 2 dependencies used so far --> `invariant` and `nullthrows`. 😄 

`rn-tester` is meant to be run as a sample application to try changes made into the `react-native` core; having 3rd party dependencies means that the new changes made into the `react-native` core will not be supported by these 3rd party dependencies. in order to avoid this unnecessary overhead usage of other dependencies is avoided.

cc @mkcode

Reply:
@princiya When I first tried to run RNTester with `yarn install`, the app didn't run for me. It only ran successfully without errors after I tried `yarn upgrade`. I'm not sure if I'm the only one having this issue, but @mkcode suggested I make a PR to update the dependencies 😄